.github/workflows/pre_merge.yaml

```diff
@@ -120,50 +120,21 @@ jobs:
     if: |
       github.event.pull_request.draft == false &&
       !(startsWith(github.event.pull_request.title, '[WIP]'))
-    runs-on: [otx-gpu-a10g-1]
+    runs-on: [self-hosted, linux, x64, dev, dmount]
```

Comment:
Can we put Intense Unit tests on the self-hosted runners as well? Or even all unit tests since you remove a label below